src/edu/csus/ecs/pc2/shadow/RemoteEventFeedMonitor.java

```diff
@@ -94,7 +96,7 @@ public class RemoteEventFeedMonitor implements Runnable {
      * @param pc2Controller an {@link IInternalController} for passing error handling back to the local PC2 system
      * @param remoteContestAPIAdapter an adapter for accessing the remote contest API
      * @param remoteURL the URL to the remote CCS
-     * @param login the login (account) on the remote CCS
+     * @param k the login (account) on the remote CCS
```

Comment:
I do not understand this change.  There is no parameter named "k" in the Constructor's argument list, and further it appears that "login" is still the correct parameter name.

Reply:
That is a mistake on my part.  Looks like I pressed a "k" instead of "ctrl-k" when I was searching for "login", and it wrote a 'k' over the login I was on.  I'll put it back to "login".

Reply:
Can this be resolved John C. ?

Reply:
I'm not sure why the fix isn't showing here in the GitHub conversation, but it does look like it was corrected in the updated copy which I pulled from the fork, so I'll mark it resolved (it's certainly not worth holding up the PR for...)